fecgov/openFEC
# Fix /candidates/totals/ COH and debt for F3&F3P combo committees

## Summary (required)

- Resolves #3635 

Fix `/candidates/totals/` cash on hand and debts owed by committee for F3&F3P combo committees. Some candidates file on F3 and F3P - this adjusts the COH calculation for `ofec_candidate_totals_vw` where there is more than one totals row for the same cycle for the same committee

Line 200: max(COH) instead of sum(COH)
Line 201: max(debt) instead of sum(debt)

## How to test the changes locally

- Set `SQLA_CONN` to DEV db
- Point `class CandidateTotal(db.Model)` to `ofec_candidate_totals_mv_tmp`
- Check COH and debts owed for Delaney on local: http://localhost:5000/v1/candidates/totals/?api_key=DEMO_KEY&sort_hide_null=false&sort_nulls_last=true&election_year=2020&cycle=2020&election_full=true&sort=-receipts&per_page=30&page=1&office=P&candidate_id=P00006213
- This should match candidate profile page and election profile page:
candidate_profile_url
https://api.open.fec.gov/v1/candidate/P00006213/totals/?cycle=2020&api_key=3WFCX5yqR4ZQmRKkQemdZVKEebxwWHQ6MPVc40l9&full_election=True

election_profile_url
https://api.open.fec.gov/v1/elections/?candidate_id=P00006213&cycle=2020&election_full=true&api_key=3WFCX5yqR4ZQmRKkQemdZVKEebxwWHQ6MPVc40l9&office=president&sort_nulls_last=true&sort=-total_receipts&per_page=100
- I also ran a candidate comparison script. Feel free to test the data in any way you like

## Impacted areas of the application
List general components of the application that this PR will affect:

- /candidates/totals/
CMS:
- Candidate totals datatable
- Candidate raising/spending charts
